test/e2e/app-dir/searchparams-reuse-loading/searchparams-reuse-loading.test.ts

```diff
@@ -181,62 +181,76 @@ describe('searchparams-reuse-loading', () => {
         let interceptRequests = false
         const browser = await next.browser(path, {
           beforePageLoad(page) {
-            page.route('**/search-params*', async (route) => {
-              if (!interceptRequests) {
-                return route.continue()
-              }
-
-              const request = route.request()
-              const headers = await request.allHeaders()
-              const url = new URL(request.url())
-              const promiseKey =
-                url.pathname + '?id=' + url.searchParams.get('id')
-
-              if (headers['rsc'] === '1' && !headers['next-router-prefetch']) {
-                // Create a promise that will be resolved by the later test code
-                let resolvePromise: () => void
-                const promise = new Promise<void>((res) => {
-                  resolvePromise = res
-                })
-
-                if (rscRequestPromise.has(promiseKey)) {
-                  throw new Error('Duplicate request')
+            page.route(
```

Comment:
This is one of those tests that we need to rewrite to use `create-router-act` instead; it's disabled in PPR temporarily.